File: Backend/Lulus.Data/Entities/ProductLine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lulus.Data.Entities
{
    public class ProductLine:BaseAttribute
    {
        public int TextureID { get; set; }
        public int ProductID { get; set; }
        public int Quantity { get; set; }
        public virtual List<OrderDetail> OrderDetails { get; set; }
        public virtual List<ProductImage> ProductImages { get; set; }
        public virtual Product Product { get; set; }
        public virtual Texture Texture { get; set; }
    }
}

File: Frontend/revolve/src/app/model/ProductLine.ts
import { ProductImage } from "./ProductImage";

export class ProductLine{
    public id: number;
    public textureName: String;
    public textureImage: String;
    public productID: number;
    public quantity: number;
    public images: ProductImage[];
    

    constructor(id:number, textureName: String, textureImage:String, productID: number, quantity: number){
        this.id = id;
        this.textureName = textureName;
        this.textureImage = textureImage;
        this.productID = productID;
        this.quantity = quantity;
        this.images = new Array<ProductImage>();
    }
    
}